nss-day-cohort-39/nutshell-razzmatazz-roadrunners
# Sg tasks

- Changes
* Now the user is able to add and delete a task.

- Test
* Click on add task button.
* fill out the fields and press save.
* Make sure the task is saved into the database and displayed in the dom.
* Click the delete button and make sure the task is deleted from the database.

- database test
{
  "tasks": [
    {
      "task": "",
      "date": "",
      "id": 
    }
  ]
}
